proto/uber/cadence/api/v1/service_worker.proto

```diff
@@ -136,11 +136,14 @@ service WorkerAPI {
   // API and return the query result to client as a response to 'QueryWorkflow' API call.
   rpc RespondQueryTaskCompleted(RespondQueryTaskCompletedRequest) returns (RespondQueryTaskCompletedResponse);
 
-  // RequestCancelWorkflowExecution is called by application worker when it wants to request cancellation of a workflow
-  // instance. It will result in a new 'WorkflowExecutionCancelRequested' event being written to the workflow history
-  // and a new DecisionTask created for the workflow instance so new decisions could be made. It fails with
-  // 'EntityNotExistsError' if the workflow is not valid anymore due to completion or doesn't exist.
-  rpc RequestCancelWorkflowExecution(RequestCancelWorkflowExecutionRequest) returns (RequestCancelWorkflowExecutionResponse);
```

Comment:
assuming its a backwards compatible

Reply:
In general it is not. However we have not released clients yet, so we can still change it.